packages/client/bin/cli.ts

```diff
@@ -79,10 +80,12 @@ const args: ClientOpts = yargs(hideBin(process.argv))
   .option('customChain', {
     describe: 'Path to custom chain parameters json file (@ethereumjs/common format)',
     coerce: (arg: string) => (arg ? path.resolve(arg) : undefined),
+    implies: 'customGenesisState',
   })
   .option('customGenesisState', {
     describe: 'Path to custom genesis state json file (@ethereumjs/common format)',
     coerce: (arg: string) => (arg ? path.resolve(arg) : undefined),
+    implies: 'customChain',
```

Comment:
Just to make sure: is this really true that both arguments always need to be used in conjunction?

Reply:
That's what the original error message said and it makes logical sense to me, that you need to provide some genesis state data for a custom chain.  [Here](https://github.com/ethereumjs/ethereumjs-monorepo/blob/0606d3cf8c316ebc5d5433c90a34632e3acefb3d/packages/client/bin/cli.ts#L754) is the original code I took this from.

Reply:
well custom genesis can be provided without custom chain --- and/or viceversa, not sure if it actually works

Reply:
The error messages indicated you can't do one without the other which I why I just copied it over. Do we want to allow one without the other?

Reply:
ok, i guess we are good for now, when we support and test then we can remove this :+1: